### DISCUSSION TITLE
Setting for related sender

### DISCUSSION THREAD

**Discussion by alexaxb** [AdhocAdam/smletsexchangeconnector#269] [Ideas]
The default behaviour when someone mails a comment regarding an existing ticket, is that the sender will be added as a related item to the ticket (besides adding the mail as a comment).
Sometimes this messes up requests with automations, that get data from a user object already added as a related item.
Maybe we could have a setting for turning off "add the sender as related item to the ticket" ?

**Comment by AdhocAdam** [comment]
I 100% most certainly know the exact type of use cases you're referring to and if there was one thing the stock connector taught me back in the day it was:

> The Related Config Items relationship can't be trusted from an automation perspective **because** of the Exchange Connector. The only reliable and probably more appropriate relationship is the other one - About Config Item/Affected Config Item. If more relationships are needed, use the Activities on the Work Item.

And as you might expect, I followed suit _because_ I wanted to ensure there was parity so people could easily onboard. Plus it made the "how" of building automation a bit more straightforward by removing the choice entirely. I confidently knew that any Related User was potentially getting notifications, but my Affected Config Items were...well the actual impacted CIs!

But I know advice like this is sometimes easier said than done given the host of automation scenarios one could configure within SCSM. I think if there was desire to move forward with this, it'd be the kind of setting that is enabled by default and you have to consciously _disable_. Maybe set the alternative relationship? Well I guess that could potentially mess things up too. Maybe pick the Relationships or none similar to a Request Offering?

Hrmm...🤔

**Reply by alexaxb** [reply]
Yea, my thought was: if this was a setting, then the default behavior would be to add the sender as related CI (as history has it), and that you need to actively have to turn it off.
